### PR TITLE
DIV-6853: Adding superuser event that may help us work around this bug

### PR DESCRIPTION
**Pull request by jvictorcf**
### Change description ###
This will allow a superuser to move the case back to the state it needs to be for issuing the AOS pack 2 again.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by johnoneill46 — APPROVED** [review]

**Review by madjava — APPROVED** [review]

File: definitions/divorce/json/CaseEvent/CaseEvent.json [hunk 1]
```diff
@@ -3083,5 +3083,17 @@
...
+  {
+    "LiveFrom": "21/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "ID": "Move_To_ServiceAppRejected",
+    "Name": "Move to service app rejected",
+    "Description": "Move to service app rejected state",
+    "DisplayOrder": 18,
+    "PreConditionState(s)": "AosAwaiting",
+    "PostConditionState": "ServiceApplicationNotApproved",
+    "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
```
Comment (yarpo) [inline definitions/divorce/json/CaseEvent/CaseEvent.json:3096]: this can be removed

Comment (jvictorcf) [reply definitions/divorce/json/CaseEvent/CaseEvent.json:3096]: Do you mean the "RetriesTimeoutURLAboutToSubmitEvent"?

**Review by shaunganley — COMMENTED** [review]
are we happy that this event will exist in PROD? can we not limit to nonprod?

File: definitions/divorce/json/CaseEvent/CaseEvent.json (hunk 1 above)
Comment (yarpo) [reply definitions/divorce/json/CaseEvent/CaseEvent.json:3096]: yes, this is completely ignored by CCD. Used only for callbacks. There is no callback here. And it's ignored anyway :)

File: definitions/divorce/json/CaseEvent/CaseEvent-temp-prod.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "LiveFrom": "21/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "ID": "Move_To_ServiceAppRejected",
+    "Name": "Move to service app rejected",
+    "Description": "Move to service app rejected state",
+    "DisplayOrder": 18,
+    "PreConditionState(s)": "AosAwaiting",
```
Comment (yarpo) [inline definitions/divorce/json/CaseEvent/CaseEvent-temp-prod.json:9]: `caseworker-divorce-superuser` has already CRU for this state

File: definitions/divorce/json/CaseEvent/CaseEvent-temp-prod.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    "PostConditionState": "ServiceApplicationNotApproved",
```
Comment (yarpo) [inline definitions/divorce/json/CaseEvent/CaseEvent-temp-prod.json:10]: `caseworker-divorce-superuser` has already CRU for this state

File: test/unit/utils/dataProvider.js [hunk 4]
```diff
@@ -33,9 +34,10 @@ module.exports = {
     ),
     CaseEvent: getCaseEventDefinitions([
       'CaseEvent',
+      'CaseEvent-bailiff-prod',
```
Comment (yarpo) [inline test/unit/utils/dataProvider.js:37]: alphabetical order

File: test/unit/utils/dataProvider.js [hunk 5]
```diff
@@ -33,9 +34,10 @@ module.exports = {
...
       'CaseEvent-resp-journey-prod',
       'CaseEvent-share-a-case-prod',
-      'CaseEvent-bailiff-prod'
+      'CaseEvent-temp-prod'
```
Comment (yarpo) [inline test/unit/utils/dataProvider.js:40]: when we don't need it it's easier to clean up.

**Comment by jvictorcf** [comment]
Closing in order to save resources while a decision is made on the solution for this.

**Pull request closed without merging**